File: Iniciante/JavaScript/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const arr = lines.shift().split(' ');
const a = parseInt(arr[0]); const b = parseInt(arr[1]); const c = parseInt(arr[2]);
var result = parseInt((a + b + Math.abs(a - b)) / 2);
var maior = parseInt((result + c + Math.abs(result - c)) / 2);

console.log(parseInt(maior)+' eh o maior');

File: Iniciante/Python/1013.py
# Ler os três valores
a, b, c = map(int, input().split())

# Encontrar o maior entre a e b
maior_ab = (a + b + abs(a - b)) // 2

# Encontrar o maior entre maior_ab e c
maior = (maior_ab + c + abs(maior_ab - c)) // 2

# Exibir o maior valor seguido da mensagem
print(f"{maior} eh o maior")
